_data/sidebars/home_sidebar.yml

```diff
@@ -21,3 +21,7 @@ children:
 - title: Contributing to LoopBack
   url: /doc/en/contrib/index.html
   output: web
+
+- title: Long-term support
+  url: /doc/en/contrib/Long-term-support.html
```

Comment:
Does this map to the previous URL that was in the `contrib_sidebar.yml`?

Reply:
Yes, it does.

My commit is moving "Long-term support" from "Contributing" sidebar to the main/home sidebar in order to give it more visibility.